src/components/Dashboard/Flavors.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { Table } from 'react-bootstrap';
+import { DashboardLayout as Layout } from 'components/Dashboard/';
+
+import { actions as flavorsActions } from 'reducers/flavors';
+import { getAllFlavors } from 'selectors/flavors';
+
+export class Flavors extends Component {
+  static propTypes = {
+    actions: PropTypes.object.isRequired,
+    opt: PropTypes.object.isRequired,
```

Comment:
Would you mind calling this `options` or `layoutOptions` maybe?

Reply:
Should be fixed in all locations, also changed defaultOpts to defaultLayoutOptions where applicable. opt is now layoutOptions